src/thor/isochrone.cc

```diff
@@ -49,13 +49,33 @@ Isochrone::Isochrone() : Dijkstras(), shape_interval_(50.0f) {
 // Construct the isotile. Use a fixed grid size. Convert time in minutes to
 // a max distance in meters based on an estimate of max average speed for
 // the travel mode.
-void Isochrone::ConstructIsoTile(
-    const bool multimodal,
-    const unsigned int max_minutes,
-    const google::protobuf::RepeatedPtrField<valhalla::Location>& locations,
-    const sif::TravelMode mode) {
+void Isochrone::ConstructIsoTile(const bool multimodal,
+                                 const valhalla::Api& api,
+                                 const sif::TravelMode mode) {
+
+  // Extend the times in the 2-D grid to be 10 minutes beyond the highest contour time.
```

Comment:
instead of computing the size of the tile outside of the isochrone api it made sense to bring all that logic in here right next to where it is needed